src/interrex_to_movebank.Rmd

```diff
@@ -29,7 +29,7 @@ metadata_colnames <- colnames(metadata)
 Set user-defined values:
 
 ```{r}
-project_id <- "HG_JUVENILE"
+project_id <- "ARMENIAN_GULL"
```

Comment:
Please don't commit changes in project selection (since this will toggle back and forth)